src/imp/span_imp.js

```diff
@@ -207,31 +183,6 @@ export default class SpanImp {
         this._joinIDs[key] = true;
     }
 
-    parent(parentSpan) {
-        if (!parentSpan) {
-            return;
-        }
-        this.setParentGUID(parentSpan.guid());
-        this._traceGUID = parentSpan.traceGUID();
-    }
-
-    span(operation) {
-        let child = new SpanImp(this._tracer);
-        child.operation(operation);
-
-        // TODO: what is the expected behavior on OpenTracing tags on
-        // child spans? The legacy Traceguide behavior relies (?) on the
-        // child spans inheriting the join IDs of the parent.
-        for (let key in this._tags) {
-            child._tags[key] = this._tags[key];
-            if (this._joinIDs[key]) {
```

Comment:
this logic should be unnecessary since explicit children will share an (OpenTracing-style) SpanContext trace_id with their parents.